python/CMakeLists.txt

```
@@ -0,0 +1,52 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+cmake_minimum_required(VERSION 3.20.1 FATAL_ERROR)
+
+# TODO: Pull this directly from the file written by rapids_cmake_write_version_file.
+set(rmm_version 22.04.00)
+
+project(
+  rmm-python
+  VERSION ${rmm_version}
+  # TODO: Building Python extension modules via the python_extension_module requires the C language
+  # to be enabled here. The test project that is built in scikit-build to verify various linking
+  # options for the python library is hardcoded to build with C, so until that is fixed we need to
+  # keep this.
+  LANGUAGES C CXX CUDA)
+
+option(FIND_RMM_CPP "Search for existing RMM CPP installations before defaulting to local files"
```

C++ for humans. CPP for parsers. :)

```suggestion
option(FIND_RMM_CPP "Search for existing RMM C++ installations before defaulting to local files"
```